packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -144,3 +144,26 @@ $ yarn add --dev esbuild@0
 To force bundling in a Docker container, set the `forceDockerBundling` prop to `true`. This
 is useful if your function relies on node modules that should be installed (`nodeModules` prop, see [above](#install-modules)) in a Lambda compatible environment. This is usually the
 case with modules using native dependencies.
+
+### Command hooks
+It is possible to run additional commands by specifying the `commandHooks` prop:
```

Comment:
Are these commands executed inside the docker container or on the host OS?

Reply:
In the container when Docker bundling is used, on the host OS with local bundling. Will make this clear in the README.